systems/cryptography/caesar-cipher/C

```diff
@@ -1 +1,23 @@
-
+#include<stdio.h>
+#include<string.h>
+int main()
+{
+int l,i,s;
+char a[100];
+printf("Enter the string:\n");
+gets(a);
+printf("enter the shift value:\n");
+scanf("%d",&s);
+l=strlen(a);
+for(i=0;i<l;i++)
+{
+if(a[i]<='z'&&a[i]>='a')
+a[i]=((a[i]-'a'+s)%(26))+'a';
+else if(a[i]<='Z'&&a[i]>='A')
+a[i]=((a[i]-'A'+s)%(26))+'A';
```

Comment:
Small comments near the main algorithmic part of the code will make it easier for others to understand the merged code.